usermods/TetrisAI_v2/readme.md

```diff
@@ -6,7 +6,7 @@ Version 1.0
 
 ## Installation 
 
-Just activate the usermod with `-D USERMOD_TETRISAI` and the effect will become available under the name 'Tetris AI'.
+Just activate the usermod with `-D USERMOD_TETRISAI` and the effect will become available under the name 'Tetris AI'. If you are running out of flash memory, use a different memory layout (e.g. https://github.com/MoonModules/WLED/blob/mdev/tools/WLED_ESP32_4MB_256KB_FS.csv).
```

Comment:
I would prefer you do not reference other repositories as their content may change without notice.
There are equivalent partitions included in this repository.

Reply:
confirmed ... the 256KB file is also in the AC repository since a few months.
see https://github.com/Aircoookie/WLED/tree/0_15/tools for all partition files.

Reply:
Cool there is one in the repo. I copied the link from the issue and missed the ones in the repo.